pkg/initializers/dataset/cache-initializer-template.yaml

```diff
@@ -0,0 +1,83 @@
+apiVersion: v1
+kind: ServiceAccount
+metadata:
```

Comment:
I would not use this template for now, and directly set the LWS template inside Python script.
Since we don't allow to change the default template, for now, I think this is unnecessary.
Also, that will allow use to remove pyyaml from the initializer dependency.